addons/auth_password_policy/models/res_config_settings.py

```diff
@@ -0,0 +1,27 @@
+from odoo import api, fields, models, _
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    minlength = fields.Integer("Minimum Length", help="Minimum number of characters passwords must contain.")
```

Comment:
what do you think of having a maxlength as well?

Reply:
FWIW, personally I really dislike a maximum length on passwords. I dislike it so much I don't think the user should even be able to configure it :D. I think historically there were some technical limitations that somehow forced maximum password lengths (see e.g. the second to last paragraph [here](https://security.stackexchange.com/a/33471)). But we don't have those in Odoo. Additionally, these days a lot of people hopefully use password managers (the only way to do passwords right IMHO) and I think they should be allowed to generate really long passwords.

Reply:
name it min_length (I know stupid topic but it is clearer).

Reply:
> what do you think of having a maxlength as well?
 
Lots of bad things.